metadata-models/src/main/pegasus/com/linkedin/common/lineage/EntityRelationship.pdl

```diff
@@ -0,0 +1,20 @@
+namespace com.linkedin.common.lineage
+
+import com.linkedin.common.AuditStamp
+import com.linkedin.common.Urn
+
+/**
+ * Downstream lineage information about a dataset including the source reporting the lineage
```

Comment:
Does this always need to be "Downstream Lineage".

The name of the model suggests it could represent _any_ type of relationship (which would be quite interesting

Reply:
Ah good catch- it is any type of relationship. This comment was copy+pasted